skbio/util/_decorator.py

```diff
@@ -13,6 +13,7 @@
 import decorator
 
 from ._exception import OverrideError
+from ._warning import DeprecationWarning as SkbioDeprecationWarning
```

Comment:
Would this make it harder for someone to ignore this warning, since ``SkbioDeprecationWarning`` isn't importable?